azure-functions/acbs-function/package.json

```diff
@@ -32,21 +32,20 @@
     "axios": "^1.7.7",
     "date-fns": "^2.30.0",
     "dotenv": "^16.4.5",
-    "durable-functions": "^3.1.0",
-    "lodash": "^4.17.21"
+    "durable-functions": "^3.1.0"
   },
   "devDependencies": {
-    "@ukef/dtfs2-common": "1.0.0",
```

Comment:
ah we were not actually using anything in common we had just imported it in?

Reply:
We were but due to base image lack of TS support this has been moved to a separate ticket where we will use `libs/common` contants.